stubs/tensorflow/tensorflow/keras/callbacks.pyi

```diff
@@ -0,0 +1,146 @@
+from collections.abc import Iterable, Mapping, Sequence
+from typing import Any, Literal
+from typing_extensions import TypeAlias
+
+import tensorflow as tf
+from tensorflow.distribute.experimental.coordinator import RemoteValue
+from tensorflow.keras import Model
+from tensorflow.keras.optimizers.schedules import LearningRateSchedule
+
+_Logs: TypeAlias = Mapping[str, object] | RemoteValue[Any] | None
```

Comment:
Hmm, while this type hint is correct it might be too strict. A lot of codebases probably never handle `RemoteValue[Any]` case as it depends on distributed strategy being used and common local training will never see it.

If mypy primer is happy I'm fine leaving it like this. A more lax type hint would `Mapping[str, object] | None | Any` known as Any trick where Any means some other types may happen but not mentioned.

Reply:
Mapping[str, object] may also be better relaxed to Mapping[str, Any]. Usually object type you know what to expect based on your model/metrics.

Reply:
I've not used this type yet, so I don't know what would be the most helpful in practice. If the strict typing might cause somewhat obscure errors in many codebases, then I suppose it might be better to go with the lax option.